crates/oxc_language_server/src/main.rs

```diff
@@ -398,6 +402,45 @@ impl LanguageServer for Backend {
 
         Ok(None)
     }
+
+    async fn execute_command(&self, params: ExecuteCommandParams) -> Result<Option<Value>> {
+        if params.command == "oxc.applyAllFixesFile" && params.arguments.len() == 1 {
+            let text_document =
+                VersionedTextDocumentIdentifier::deserialize(params.arguments.first().unwrap())
+                    .unwrap();
+            let uri = text_document.uri;
+            if let Some(value) = self.diagnostics_report_map.get(&uri.to_string()) {
+                join_all(value.iter().filter(|report| report.fixed_content.is_some()).map(
+                    |report| async {
+                        let fixed_content = report.fixed_content.as_ref().unwrap();
+                        return self
+                            .client
+                            .apply_edit(WorkspaceEdit {
+                                #[expect(clippy::disallowed_types)]
+                                changes: Some(std::collections::HashMap::from([(
```

Comment:
I still stink we do. 

Appling all fixes to a file and then rollbacking with CTRL + Z only works with one fix at the time.  
For me this is fine for the moment <3 thanks u

Reply:
I understand now. Ya I don’t think doing this is trivial enough to include in this PR, but I do think it would be a worthwhile change.

Going to resolve this comment as it won’t be addressed here.